qiskit_nature/drivers/second_quantization/gaussiand/__init__.py

```diff
@@ -39,7 +39,16 @@
 
 Part of this interfacing code --- specifically, the Fortran file `qcmatrixio.F` --- requires
 compilation to a Python native extension. However, Qiskit comes with pre-built binaries
-for most common platforms. If there is no pre-built binary matching your platform, then it will be
+for most common platforms.
+
+When using the existing Windows pre-built binary with anaconda, the library `libifcoremd.dll`
+is required in order to load it. In case it is missing, it can be installed as follows:
```

Comment:
Can we make this a more general statement about the Windows extension needing this dependency, which it does, no matter how you run the code i.e its more than when using it with Anaconda. If its missing and you are using Anaconda then indeed that will install it.